notebooks/official/workbench/spark/spark_ml.ipynb

```diff
@@ -0,0 +1,1623 @@
+{
```

Comment:
<p>There's a built in library for this in python that we prefer instead https://docs.python.org/3/library/uuid.html</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/838/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>

Reply:
<p>This is code from the template :/ Hyunuk maybe check with Ivan?</p>

Reply:
<p>It's copied over from the template but I'll post an issue on the repo.</p>

Reply:
<p>Created an <a href="https://github.com/GoogleCloudPlatform/vertex-ai-samples/issues/851" target="_blank">issue</a> . Will discuss with Ivan!</p>

Reply:
<p>I looked the history and the repo already had <code>uuid.uuid4()</code>  before, but since the function creates random uuid with dash(-), which is not accepted in the name of BigQuery dataset. I still think using uuid package and substring it like <code>uuid = uuid[:8]</code>  is simpler...</p>